src/runtime/hexagon/rpc/simulator/session.cc

```diff
@@ -1311,6 +1311,8 @@ detail::Optional<HEXAPI_Nullptr> SimulatorRPCChannel::to_nullptr(const detail::M
 
 TVM_REGISTER_GLOBAL("tvm.contrib.hexagon.create_hexagon_session")
```

Comment:
Can we include a test that we expect to fail when the stack size is too small, and expect to pass when the stack size is set larger?

Reply:
The simulator ignores the stack size at the moment.

Reply:
https://github.com/apache/tvm/pull/11046